ep_gui/resources/lang/ep_gui.en-US.rc

```diff
@@ -168,8 +168,8 @@ BEGIN
     IDS_START_MAXFREQAPPS_0     "None"
     IDS_START_MAXFREQAPPS_6     "6 (default)"
     IDS_START_MAXFREQAPPS_99999 "Unlimited"
-    IDS_START_MONITOROVERRIDE_L1 "When using multiple displays, open Start on this monitor when invoked using"
-    IDS_START_MONITOROVERRIDE_L2 "the keyboard"
+    IDS_START_MONITOROVERRIDE_L1 "When using multiple displays, open Start-menu on this monitor when invoked"

```

Comment:
"Start Menu" should not be dashed.

Reply:
I am not born English and certainly Start Menu is more appropriate